### ISSUE TITLE
Deprecations to fix.

### ISSUE THREAD

**Issue by johnunclesam** [ember-cli-deploy/ember-cli-deploy#465]
```
DEPRECATION: `ember-cli/ext/promise` is deprecated, use `rsvp` instead. Required here:
  Object.<anonymous> (node_modules\ember-cli-deploy-build\index.js:4:15)

DEPRECATION: `ember-cli/ext/promise` is deprecated, use `rsvp` instead. Required here:
  Object.<anonymous> (node_modules\ember-cli-deploy-revision-data\index.js:4:15)
```

**Comment by lukemelia** [comment]
@johnunclesam Thanks for the report. We've got PRs fixing these pending in both of those plugin repos. Just need to figure out our strategy for node version support and semver and we'll get plugin releases out that will remove these deprecation warnings.

**Comment by lukemelia** [comment]
Now released as 1.0.0-beta releases across the plugins.

**Comment by lukemelia** [comment]
Core plugins have been released as 1.0.0 (as of a few days ago).
